src/utils/getTaskIdHistory.js

```diff
@@ -0,0 +1,5 @@
+import { RECENT_TASKS_STORAGE_KEY } from './constants';
+
+export default JSON.parse(
```

Comment:
This is definitely something where you want to use [dexie](http://dexie.org/) as you can store these as strings in a table instead of needing to do the serialization dance.

Reply:
Interesting. Do you prefer dexie over localforage? We should probably use only one indexDB library wrapper.

Reply:
They each have their purpose. localforage is good for doing key-value stores like localstorage, where you just need to look up a single value. dexie is good for collections of data, like capturing a list.